PopulateTools/gobierto
# Remove translated routes

Connects to #112

### What does this PR do?

This PR removes translated routes support. In the public resources, we have picked Spanish as main language.

### How should this be manually tested?

If you visit budgets visualizations or budgets consultations you'll see the routes main segments in Spanish.



Routes paths are defined in Spanish for the time being
